Entity/View.php

```diff
@@ -93,9 +93,9 @@ public function __construct()
     public function __clone()
     {
         if ($this->id) {
-            unset($this->id);
-            unset($this->created);
```

Comment:
Violation of phpstan, as the finalizedBy cannot be null. As discussed we will set a new DateTime to Now() for these two fields (created & modified). (Can you do this for this pull request).
I will take care of the tests for the "id" field on __clone of an object.

Reply:
see 274ae5530cc8872da24eec4e7169c1cf23d0709b

Indeed Doctrine i clever enough